index.html

```diff
@@ -122,6 +122,9 @@ <h3>関連リンク</h3>
   <!-- FOOTER  -->
   <div id="footer_wrap" class="outer">
     <footer class="inner">
+
+      <img src="./images/rogo_color.png" class="img_logo" >
```

Comment:
- [ ] typo `rogo` -> `logo` .

Reply:
00274f368835d3751daec600ddd6d3e07c8ee264
Fixed typo from logo to rogo.